test/models/email_configuration_test.rb

```diff
@@ -128,7 +128,7 @@ def global_settings
       password: "global-password",
       enable_starttls_auto: true,
       enable_starttls: false,
-      tls: true,
+      openssl_verify_mode: true
```

Comment:
IIRC the verify mode shouldn't be `true` but there were some OpenSSL constants.

Reply:
See https://guides.rubyonrails.org/action_mailer_basics.html
Probably you want `OpenSSL::SSL::VERIFY_PEER`

Reply:
Yeah, you're right :sweat_smile:  `mode == true` is quite stupid :grimacing:

Reply:
@akostadinov yeah, that doc says:
```
You can use the name of an OpenSSL verify constant ('none' or 'peer') or directly the constant (OpenSSL::SSL::VERIFY_NONE or OpenSSL::SSL::VERIFY_PEER).
```

There is a conversion for https://github.com/mikel/mail/blob/3dd53e046024f36cb6bfec57a6a131c45e0121fa/lib/mail/network/delivery_methods/smtp.rb#L179-L181

Reply:
I wonder how did it work with `true`...

Reply:
I see now. That's fine.